test/mocha/integ/handlers/router-entities/route-caching/dynamo-route-caching-provider.test.ts

```diff
@@ -152,13 +156,63 @@ const TEST_UNCACHED_ROUTES = new CachedRoutes({
 })
 
 describe('DynamoRouteCachingProvider', async () => {
+  let spy: SinonSpy
+
+  beforeEach(() => {
+    spy = sinon.spy(metric, 'putMetric')
+  })
+
+  afterEach(() => {
+    spy.restore()
+  })
+
   setupTables(TEST_ROUTE_CACHING_TABLE, TEST_ROUTE_DB_TABLE)
   const dynamoRouteCache = new DynamoRouteCachingProvider({
     routesTableName: DynamoDBTableProps.RoutesDbTable.Name,
     routesCachingRequestFlagTableName: DynamoDBTableProps.RoutesDbCachingRequestFlagTable.Name,
     cachingQuoteLambdaName: 'test',
   })
 
+  it('Cached routes hits new cached routes lambda', async () => {
+    spy.withArgs('CachingQuoteForRoutesDbRequestSentToLambdanewcachinglambda', 1, MetricLoggerUnit.Count)
```

Comment:
@cgkol @xrsv this is the best I can test assert against hitting new caching routing lambda - by spying the new metric I added, and ensure to set up the cached routes retrieval, so that it sends a async routing lambda invocation.